pkg/kyma/cmd/provision/minikube/hosts_windows.go

```diff
@@ -4,14 +4,34 @@ package minikube
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/kyma-project/cli/internal/step"
 )
 
 func addDevDomainsToEtcHostsOSSpecific(o *MinikubeOptions, s step.Step, hostAlias string) error {
 
 	s.LogErrorf("Please add these lines to your " + hostsFile + " file:")
-	fmt.Println(hostAlias)
+	hostsArray := strings.Split(hostAlias, " ")
+	ip := hostsArray[0]
+	hostsPerLine := split(hostsArray[1:], 7) // 7 hostnames per line
+
+	for k := 0; k < len(hostsPerLine); k++ {
```

Comment:
This looks like a lot of code to do a very simple splitting of slices. You can achieve the same with the following code without the `split` function at all:
```go
hostsArray = hostsArray[1:]
for len(hostsArray) > 0 {
	chunkLen := 7
	if len(hostsArray) < chunkLen {
		chunkLen = len(hostsArray)
	}
	fmt.Printf("%s\n", strings.Join(hostsArray[:chunkLen], " "))
	chunks = chunks[chunkLen:]
}
```

Basically you strip out and print chunks of your desired size until the slice is empty.